webapp/src/actions.ts

```diff
@@ -369,7 +372,7 @@ export function incomingCallOnChannel(channelID: string, callID: string, callerI
             await dispatch(getProfilesByIdsAction([callerID]));
         }
 
-        await dispatch({
```

Comment:
So dispatching is always synchronous or there's more to it?

Reply:
Nope, dispatching is async, it's just that putting await dispatch at the return is useless/redundant (since the function returns the promise immediately, so awaiting it does nothing).